File: events-bot/discord.py
import discord
from discord.ext import commands

bot = commands.Bot(command_prefix='!')

@bot.event
async def on_ready():
    print('Bot is ready.')

@bot.command()
async def create_event(ctx, event_name: str, event_time: str):
    event_channel = bot.get_channel(channel_id)
    event_message = await event_channel.send(f'Event "{event_name}" has been created at {event_time}.')
    await event_message.add_reaction('👍')
    await event_message.add_reaction('👎')

bot.run('your_bot_token_goes_here')

File: events-bot/discord.js
const Discord = require('discord.js');
const client = new Discord.Client();

client.on('ready', () => {
    console.log(`Logged in as ${client.user.tag}!`);
});

client.on('message', msg => {
    if (msg.content === '!create_event') {
        let event_name = msg.content.split(" ")[1];
        let event_time = msg.content.split(" ")[2];
        let event_channel = msg.guild.channels.cache.get(channel_id);
        event_channel.send(`Event "${event_name}" has been created at ${event_time}.`)
            .then(message => {
                message.react('👍');
                message.react('👎');
            });
    }
});

client.login('your_bot_token_goes_here');
